CVLByExamples/README.md

```diff
@@ -0,0 +1,15 @@
+
+
+These examples can serve as an introduction to CVL ( Certora Verification Language ).
+It also contains examples that are based on real-life code and the spec that have caught critical bugs.
+
+See https://docs.certora.com for a complete guide. 
```

Comment:
I would link specifically to the guide that covers this example

Reply:
big todo